src/web/app/components/question-responses/grouped-responses/grouped-responses.component.ts

```diff
@@ -25,6 +25,7 @@ export class GroupedResponsesComponent extends InstructorResponsesViewBase imple
   @Input() responses: QuestionOutput[] = [];
   @Input() userToEmail: Record<string, string> = {};
 
+  @Input() isLastResponse: boolean = false;
```

Comment:
We should not call it as last response as clearly this is grouped responses. Maybe `lastGroupResponses`